Amethyst/default.amethyst

```diff
@@ -190,6 +190,7 @@
         "key": "x"
     },
     "floating": [],
+    "floating-is-blacklist": false,
```

Comment:
This should probably be true by default as that is the existing behavior.

Reply:
Updated